source/Calamari.Tests/KubernetesFixtures/HelmUpgradeFixture.cs

```diff
@@ -95,7 +95,7 @@ public virtual void SetUp()
             Environment.SetEnvironmentVariable("TentacleJournal",
                                                Path.Combine(StagingDirectory, "DeploymentJournal.xml"));
 
-            Variables = new VariablesFactory(FileSystem).Create(new CommonOptions("test"));
+            Variables = new VariablesFactory(FileSystem, new SilentLog()).Create(new CommonOptions("test"));
```

Comment:
Is it worth passing in `ConsoleLog` instead of `SilentLog`? Would that get us more useful output on a test failure?

Reply:
these logs would have been previously just consumed, so to avoid having to rethink about these tests im leaving behavior as-is for now.